### PR TITLE
Test banner video without preload

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <85789734+debo19@users.noreply.github.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit fa855360e6a350afaac47b89314ccf7b595301e3 at: https://62e980fe9f86095b2034cdd1--layer5.netlify.app

**Comment by leecalcote** [comment]
It looks like a single load (in Safari).

**Comment by debo19** [comment]
Still not working on chrome.

**Comment by leecalcote** [comment]
Are you sure that both instances are the full MBs?

**Comment by leecalcote** [comment]
I do see a prefetch, but only see one full load of the video file.

File: src/sections/Home/Banner-4/index.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const Banner1 = () => {
         </Col>
         <Col sm={4} lg={6} className="section-title-wrapper video-col">
           <Link to="/cloud-native-management/meshmap">
-            <video autoPlay muted loop preload="metadata" className="meshmapVideo">
```
Comment (Nikhil-Ladha) [inline src/sections/Home/Banner-4/index.js:43]: So, the `preload` is the culprit for the file being downloaded twice? 🤔

**Comment by l5io** [comment]
🚀 Preview for commit 5e8f7f1f908817fe3ceb8df50b9dbe55b9de8560 at: https://62ea41d2f354f0061bd15053--layer5.netlify.app

**Pull request closed without merging**